app/helpers/layout_helper.rb

```diff
@@ -45,7 +45,9 @@ def fetch_user
   end
 
   def index_pages
-    []
+    [
+      { path: '/next_architectures', props: { path: architectures_path } },
```

Comment:
the paths that are registered here will be added to React-router when the app starts,
and the props will be passed to the index page template.

Reply:
This should also use the helper as mentioned [here](https://github.com/theforeman/foreman/pull/8596/files#r653379610).